src/controllers/profile-settings.controller.js

```diff
@@ -50,6 +51,7 @@ class ProfileSettingsController {
       );
     }));
     const userData = await UserService.findUserByProperty({ email: userToken.email });
+    await RequestService.updateRequest({ requesterId: userToken.id }, { requesterPicture: `${req.body.profilePicture}` });
```

Comment:
@erickyvand Any reason why this PR is changing the update profile functionality? From the description, the PR is supped to return a trip id when commenting on a trip request.

Reply:
There were lines of code that @higustave-ops wanted to include and because his PR was merged, and I had a pending PR so he told me to add those line of code